### DISCUSSION TITLE
mp3 audio stream as input

### DISCUSSION THREAD

**Discussion by unosonic** [mcguirepr89/BirdNET-Pi#378] [Ideas]
Hello,
having the possibility to source from a RTSP stream instead of a microphone is great, however it would also be helpful to grab the audio from an Icecast-server in mp3 (or ogg) format. Would that be possible?
bests,

**Comment by mcguirepr89** [comment]
Hi, @unosonic 

As it is, you can use the HTTP(s) protocol for the stream as well, so you can utilize an IceCast2 stream instead of an RTSP. My best!

**Reply by unosonic** [reply]
hi Patrick, it works. Great, thanks! I should have tested it before asking questions...
